localstack/services/sqs/provider.py

```diff
@@ -571,7 +571,8 @@ def on_before_stop(self):
         self._queue_update_worker.stop()
         self._stop_cloudwatch_metrics_reporting()
 
-    def _require_queue(self, account_id: str, region_name: str, name: str) -> SqsQueue:
+    @staticmethod
+    def _require_queue(account_id: str, region_name: str, name: str) -> SqsQueue:
```

Comment:
Because this is now a static method, the mutex is made a global variable `_STORE_LOCK`. This also makes sense as it is not bound to a Provider instance but the Store instance.